File: Practicas/Marco-Oregel/Practica3/Book.py
class Book:

    def __init__(self,title,author):
        self.title = title
        self.author = author
        self.isBorrowed = False

    def description(self):
        return f"- {self.title} by {self.author} - {"Borrowed" if self.isBorrowed else "Available"}"
        

File: Practicas/Yahir-Hermosillo/Parcial3/Practica4/Book.java
// Book.java
public class Book {
    private String title;
    private String author;
    private boolean isBorrowed;
    
    public Book(String title, String author) {
        this.title = title;
        this.author = author;
        this.isBorrowed = false;
    }
    
    public String getTitle() {
        return title;
    }
    
    public String getAuthor() {
        return author;
    }
    
    public boolean isBorrowed() {
        return isBorrowed;
    }
    
    public void setBorrowed(boolean borrowed) {
        isBorrowed = borrowed;
    }
    
    public String description() {
        return title + " por " + author + " - " + (isBorrowed ? "Prestado" : "Disponible");
    }
}